docs/hugo/content/contributing/_index.md

```diff
@@ -11,56 +11,6 @@ cascade:
 description: "How to contribute new resources to Azure Service Operator v2"
 ---
 
-## Related pages
```

Comment:
I feel like we should somehow be linking or highlighting the developer setup page here at least? It's IMO the most important page for people to see. More important even than Directory structure of the operator (which is now the top thing here on the index).

Wondering if we should split the contributing section up entirely and move the other bits to subpages as well and just have index be a TOC-esque thing that links to the various sub-pages?

Reply:
I've restored the `Related Pages` section, for now. 

It feels out of place to me, especially given we already have a list of related pages in the lefthand sidebar, plus a table of content in the righthand sidebar. That said, I think we need to restructure this area and I'm happy to leave the section in until we do that.